### ISSUE TITLE
Add Comments

### ISSUE THREAD

**Issue by kata-gatame** [The-Expanse-Discord/eslint-config-expanse#1]
Add comments throughout the module.

**Comment by kata-gatame** [comment]
Markdown is proving to be a bit problematic for displaying some of this information.  Will transition to HTML.

**Comment by kata-gatame** [comment]
Once the official refinement is merged into master, update the 'setting' column with line number links.

This will cut down on how crunched the tables are.
